Fixed export formats for empty project (#5899)
opencv/cvat: CHANGELOG.md

- Invalid mask when running automatic annotation on a task (<https://github.com/opencv/cvat/pull/5883>)
- Cloud storage content listing when the manifest name contains special characters
  (<https://github.com/opencv/cvat/pull/5873>)

> - Export formats for empty project are not shown (<https://github.com/opencv/cvat/pull/5899>)

Reviewer: ```suggestion
- Empty list of export formats for a project without tasks (<https://github.com/opencv/cvat/pull/5899>)
```
